docs/core/testing/unit-testing-code-coverage.md

```diff
@@ -0,0 +1,406 @@
+              </conditions>
+            </line>
+            <line number="17" hits="4" branch="False" />
+            <line number="18" hits="4" branch="False" />
+            <line number="20" hits="3" branch="False" />
+            <line number="21" hits="4" branch="False" />
+            <line number="23" hits="11" branch="False" />
+          </lines>
+        </class>
+      </classes>
+    </package>
+  </packages>
+</coverage>
+```
+
+### MSBuild
+
+As an alternative, you could use the MSBuild package if your build system already makes use of MSBuild. From a command prompt, change directories to the *XUnit.Coverlet.MSBuild* project, and run the `dotnet test` command:
+
+```dotnetcli
+dotnet test /p:CollectCoverage=true
+```
+
+Below is the example *coverage.json* file from the corresponding sample repo.
+
+```json
+{
+  "Numbers.dll": {
+    "Numbers\\PrimeService.cs": {
+      "Numbers.PrimeService": {
+        "System.Boolean Numbers.PrimeService::IsPrime(System.Int32)": {
+          "Lines": {
+            "8": 11,
+            "9": 11,
+            "10": 3,
+            "11": 3,
+            "14": 22,
+            "15": 7,
+            "16": 7,
+            "17": 4,
+            "18": 4,
+            "20": 3,
+            "21": 4,
+            "23": 11
+          },
+          "Branches": [
+            {
+              "Line": 9,
+              "Offset": 7,
+              "EndOffset": 9,
+              "Path": 0,
+              "Ordinal": 0,
+              "Hits": 3
+            },
+            {
+              "Line": 9,
+              "Offset": 7,
+              "EndOffset": 14,
+              "Path": 1,
+              "Ordinal": 1,
+              "Hits": 8
+            },
+            {
+              "Line": 14,
+              "Offset": 57,
+              "EndOffset": 18,
+              "Path": 1,
+              "Ordinal": 5,
+              "Hits": 7
+            },
+            {
+              "Line": 16,
+              "Offset": 27,
+              "EndOffset": 29,
+              "Path": 0,
+              "Ordinal": 2,
+              "Hits": 4
+            },
+            {
+              "Line": 16,
+              "Offset": 27,
+              "EndOffset": 34,
+              "Path": 1,
+              "Ordinal": 3,
+              "Hits": 3
+            },
+            {
+              "Line": 14,
+              "Offset": 57,
+              "EndOffset": 59,
+              "Path": 0,
+              "Ordinal": 4,
+              "Hits": 4
+            }
+          ]
+        }
+      }
+    }
+  }
+}
+```
+
+> [!NOTE]
+> There is a subtle difference between the outputs of the two `dotnet test` commands. The first output was an XML file, whereas the last is a JSON file.
+
+## Use with .NET global tooling
+
+To use coverlet globally, it can be installed as a [.NET global tool](../tools/global-tools.md). Optionally, run the following [`dotnet tool install`](../tools/dotnet-tool-install.md) command:
```

Comment:
To use globally -- that is, without installing the package in every test project?

Reply:
That is how it would work, yes.